wikipedia/wikipedia.py

```diff
@@ -716,8 +719,10 @@ def _wiki_request(params):
   '''
   global RATE_LIMIT_LAST_CALL
   global USER_AGENT
+  global VARIANT
 
   params['format'] = 'json'
+  params["variant"] = VARIANT
```

Comment:
`variant` is not actually a valid parameter [`query`](https://www.mediawiki.org/wiki/API:Query) or any of the other MediaWiki API options as far as I can find.